src/Plugin/VoteUpDownWidget/Alternate.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Drupal\vud\Plugin\VoteUpDownWidget;
+
+use Drupal\vud\Plugin\VoteUpDownWidgetBase;
+
+/**
+ * Assigns ownership of a node to a user.
+ *
+ * @VoteUpDownWidget(
+ *   id = "alternate",
+ *   label = @Translation("Alternate rating widget")
+ *  )
+ */
+class Alternate extends VoteUpDownWidgetBase {
+
+  public function build() {
```

Comment:
This method is declared on every widget, sounds like a good candidate to be moved to the parent class.
Any reason not to declare it there?
Especially after having a `getWidgetTemplate()` the only widget specific line passing can be translated to `'#widget_template' => $this->getWidgetTemplate()`.